File: integration-tests/mongodb-panache/src/main/java/io/quarkus/it/mongodb/panache/book/BookDetail.java
package io.quarkus.it.mongodb.panache.book;

public class BookDetail {

    private String summary;

    private int rating;

    public String getSummary() {
        return summary;
    }

    public BookDetail setSummary(String summary) {
        this.summary = summary;
        return this;
    }

    public int getRating() {
        return rating;
    }

    public BookDetail setRating(int rating) {
        this.rating = rating;
        return this;
    }
}

File: integration-tests/mongodb-panache-kotlin/src/main/kotlin/io/quarkus/it/mongodb/panache/book/BookDetail.kt
package io.quarkus.it.mongodb.panache.book

class BookDetail {
    var summary: String? = null
        private set

    var rating = 0
        private set

    fun setSummary(summary: String?): BookDetail {
        this.summary = summary
        return this
    }

    fun setRating(rating: Int): BookDetail {
        this.rating = rating
        return this
    }
}
